src/src/index.ts

```diff
@@ -7,14 +7,19 @@ import './loader'
 import polishApi from './polish-api'
 import { security } from './security'
 import web from './web'
+import jwtSigner from './jwt-signer'
+import jwtVerifier from './jwt-verifier'
+
 const trace = debug('aspsp-mock')
 const port = process.env.LISTEN_PORT || 3000
 const app = express()
 
+app.use(jwtSigner)
 app.use(security)
 app.use('/api', consent)
 app.use('/data', data)
 app.use(polishApi)
 app.use(web)
+app.use(jwtVerifier)
```

Comment:
I think You should hook jwtVerifier before PolishAPI router to apply it to the operations